packages/connect-express/src/express-connect-middleware.ts

```diff
@@ -74,7 +74,7 @@ export function expressConnectMiddleware(
     res: express.Response,
     next: express.NextFunction
   ) {
-    const uHandler = paths.get(req.url);
+    const uHandler = paths.get(req.url.split("?", 2)[0]);
```

Comment:
I think a short comment explaining that this strips the query from the request path will be helpful for future us 🙂

Reply:
Done.